email/1.3/modules/ROOT/edits/email-trigger.adoc

```diff
@@ -0,0 +1,219 @@
+
+The following screenshot shows this parameter configuration for the source:
+
+.Enable Watermark with On New Email - IMAP source
+image::email-watermark-1.png[Enable Watermark with On New Email - IMAP]
+
+In the XML editor, the configuration `watermark="true"` is under the `listener-imap` tag:
+
+[source, xml, linenums]
+----
+<flow name="OnNewEmail-IMAP">
+    <email:listener-imap config-ref="Email_IMAP" watermark="true">
+        <scheduling-strategy>
+            <fixed-frequency/>
+        </scheduling-strategy>
+    </email:listener-imap>
+</flow>
+----
+
+== Configure IMAP Matcher and POP3 Matcher Parameters
+
+Email Connector *On New Email - IMAP* and *On New Email - POP3* sources support *Imap matcher* and *Pop3 matcher* parameters that enable you to configure custom matchers  and provide another way of filtering dispatched messages. Each source matcher can filter emails by dates, addresses, subject, flags, and so on.
+
+The following example shows how to configure these parameters in Anypoint Studio by dispatching all messages that contain the word `IMPORTANT`:
+
+. Select the *On New Email - IMAP* or *On New Email - POP3* source in the Studio canvas.
+. On the source configuration screen, for the *Imap matcher* or *Pop3 matcher* parameters, select either:
+
+* *Edit inline* +
+Set all the inline fields *Seen*, *Answered*, *Deleted*, *Recent*, and so on.
+* *Global reference* +
+Create a global reference configuration to set all the inline fields.
+
+[start=3]
+. In the *Subject regex* field, set the value as `IMPORTANT`.
+
+The following screenshot shows the *Imap matcher*  edit inline configuration for the *On New Email - IMAP* source:
+
+.Imap matcher with On New Email - IMAP source
+image::email-imap-matcher.png[Imap matcher with On New Email - IMAP]
+
+In the XML editor, the configuration `imap-matcher subjectRegex="IMPORTANT"` is under the `listener-imap` tag:
+
+[source, xml, linenums]
+----
+<flow name="OnNewEmail-IMAP">
+    <email:listener-imap config-ref="Email_IMAP">
+        <scheduling-strategy>
+            <fixed-frequency/>
+        </scheduling-strategy>
+        <email:imap-matcher subjectRegex="IMPORTANT"/>
+    </email:listener-imap>
+</flow>
+----
+
+== Configure Attachment Naming Strategy Parameter
+
+Email Connector *On New Email - IMAP*, *On New Email - POP3*, *List - IMAP*, and *List - POP3* operations support the *Attachment naming strategy* parameter, which enables you to specify a strategy for naming attachments. You can set the parameter at the global element or override it at the operation level. The parameter has the following options to choose from:
+
+* *NAME (Default)* +
+Email Connector names the attachment after the value of the `filename` field in the `Content-Disposition` line of the attachment (this is the standard and recommended option to set the name for an attachment). If the `filename` field is not present or is empty, then the attachment is named `Unnamed`.
```

Comment:
I think that identifying this as the default assumes the parenthetical statement:
```suggestion
Email Connector names the attachment after the value of the `filename` field in the `Content-Disposition` line of the attachment. If the `filename` field is not present or is empty, then the attachment is named `Unnamed`.
```